contracts/Strategy.sol

```diff
@@ -270,20 +192,32 @@ contract Strategy is BaseStrategy {
         // claim interest from lending
         _takeLendingProfit();
```

Comment:
Removed in 1a30f4e90adce0b373c023fe87cc087dda92ec88.

If I understand correctly this is already covered by the call to `liquidatePosition` below, making `_takeLendingProfit` unnecessary. 

I will leave the conversation open instead of marking it as resolved so you can confirm, since this may be a sensible change.

Reply:
sorry I just reread you were asking for my confirmation: 

yes. the lending profit is accounted for in the same way an airdrop would. so this LGTM